.clj-kondo/hooks/common.clj

```diff
@@ -200,6 +200,34 @@
                 body))]
     {:node node*}))
 
+(defn- let-second-inner [body bindings]
+  (let [binding-infos (for [[model {[binding value] :children}] (partition 2 bindings)]
+                        {:model   model
+                         :binding binding
+                         :value   (or value
+                                      (hooks/token-node 'nil))})]
+    (-> (hooks/vector-node
+         [(hooks/vector-node (map :model binding-infos))
+          (-> (hooks/list-node (list* (hooks/token-node `let)
+                                    (hooks/vector-node (mapcat (juxt :binding :value) binding-infos))
+                                    body))
+              (with-meta (meta body)))])
+        (with-meta (meta body)))))
+
+(defn let-second
+  "Helper for macros that have a shape like, where the let is for second arg.
+
+    (my-macro x y
+    ...)
+
+    =>
+
+    (let [y nil]
+    ...)"
```

Comment:
This example doesn't seem to match `with-link-card-fixture-for-dashboard`.

I think it should be like `(my-macro x [y]) => (let [y nil])`